go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/PagerDuty/go-pagerduty v1.5.1
 	github.com/PuerkitoBio/goquery v1.8.0
 	github.com/adamliesko/retry v0.0.0-20200123222335-86c8baac277d
+	github.com/antlr/antlr4/runtime/Go/antlr v0.0.0-20220209173558-ad29539cd2e9
```

Comment:
was this dep added by accident? I don't see it being imported in this PR

Reply:
I have no idea, I just run the `go mod tidy` and this should be updated automatically.